.github/workflows/main.yml

```diff
@@ -2078,7 +2078,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        graal: [ '20.3.0', '21.1.0' ]
+        graal: [ '20.3.8', '21.3.3.1' ]
```

Comment:
bumped the GraalVM versions to latest update release. Should 22 be also in the matrix?